style.css

```diff
@@ -46,28 +48,28 @@ html, body {
     float: right;
     z-index: 4;
     display: flex;
-    border-right: 2px solid #000000;
-    margin-right: 15px;
+    border-right: 2px solid var(--main-text-color);
```

Comment:
why you use text colors to describe borders?

Reply:
Renamed it